index.html

```diff
@@ -79,22 +79,7 @@
 
 			<div id='map'></div>
 
-<!--commenting out the layer switcher for now
-			<div class='legend'>
-				<div class='legend-title'>Click on a layer name below to toggle its visibility</div>
-				<div class='legend-scale'>
-					<ul class='legend-labels'>
-						<li onClick="showHideLayer('people-points', markerNames=['people_locations']);"><span id="raising_blended_learners_campuses"></span>People</li>
-						<li onClick="showHideLayer('raising-school-leaders-points', markerNames=['raising_school_leaders']);"><span id="raising_school_leaders"></span>Places</li>
-					</ul>
-
-
-				</div> <!-- end of div class='legend-scale' -->
```

Comment:
This code has been commented out for some time and seems unneeded in this project